packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" setup>
+import { createTooltip } from "@sit-onyx/headless";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import type { OnyxTooltipProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxTooltipProps>(), {
+  color: "neutral",
+  position: "top",
+  fitParent: false,
+});
+
+defineSlots<{
+  /**
+   * Default slot where the parent content is placed that should open/close the tooltip.
+   */
+  default(props: Record<string, unknown>): unknown;
+}>();
+
+const {
+  elements: { trigger, tooltip },
+} = createTooltip({});
+</script>
+
+<template>
+  <div class="onyx-tooltip-wrapper">
+    <div
+      v-bind="tooltip"
+      class="onyx-tooltip onyx-text--small onyx-truncation-multiline"
+      :class="{
+        'onyx-tooltip__danger': props.color === 'danger',
+        'onyx-tooltip--bottom': props.position === 'bottom',
+        'onyx-tooltip--fit-parent': props.fitParent,
+      }"
+    >
+      <OnyxIcon v-if="props.icon" :icon="props.icon" size="16px" />
+      <span>{{ props.text }}</span>
+    </div>
+
+    <div v-bind="trigger">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+.onyx-tooltip {
+  --background-color: var(--onyx-color-base-neutral-900);
```

Comment:
if the background-color can be easily overwritten with this local css variable, shouldn't we do the same for the "color"?